app/src/main/dist/config/crate.yml

```diff
@@ -866,11 +866,11 @@
 
 #license.enterprise: true
 
-# To use any of the modules in this section, Crate.io must have given you
-# permission to enable and use such Enterprise Features (see
+# To enable or use any of the settings in this section, Crate.io must have
```

Comment:
I would follow @dobe s comment and put it like `...any of the enterprise settings..` to get rid of `section`.